daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/svalue/EqualitySpec.scala

```diff
@@ -1,470 +0,0 @@
-// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Why can we delete this whole test suite?

Reply:
This is tested extensively in [ComparisonSBuiltinTest](https://github.com/digital-asset/daml/blob/master/daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/ComparisonSBuiltinTest.scala)